Assets/Scripts/ListComponents/TasksView/TasksUIController.cs

```diff
@@ -1,3 +1,29 @@
+using UnityEngine;
+using UnityEngine.Scripting;
+
 public class TasksUIController : ListUIController<Task>
 {
+    private GameObject newTaskButton;
```

Comment:
this can probably be in the parent class since both tasks and rewards have a new list item button.

Reply:
changed